### DISCUSSION TITLE
[公告]关于绳网遭遇不明黑客攻击公告

### DISCUSSION THREAD

**Discussion by Xmug** [share121/inter-knot#730] [其他]
亲爱的绳匠、盗洞客、中间人们，很遗憾刚刚绳网遭遇了一场不明的黑客攻击，该次攻击非常强力，一度使绳网无法打开，但是绳网团队最终解决了该次攻击，基于情报共享的原则，我们决定将本次防御的过程与细节展现出来。
以下是视频：[视频点我](https://www.bilibili.com/video/BV1bi421h79s)

**Comment by Xinyer-team** [comment]
省流：诈骗

**Comment by aiTunes-dev** [comment]
每天上一当，当当不重样（

**Comment by niegong** [comment]
又被骗了

**Comment by yang1cong2** [comment]
[]~(￣▽￣)~*

**Comment by GanlinSama** [comment]
/(ㄒoㄒ)/~~

**Comment by YusuKaslana** [comment]
每天上一当，当当不一样

**Comment by Lephaethon** [comment]
又被骗了
